pkg/volume/volume.go

```diff
@@ -129,12 +132,41 @@ type Deleter interface {
 // Attacher can attach a volume to a node.
 type Attacher interface {
```

Comment:
Just realized none of the plugins actually implement the Attacher interface yet. They all just overload the Mounter code. Nice.